src/output/icons.rs

```diff
@@ -139,11 +148,15 @@ const DIRECTORY_ICONS: Map<&'static str, char> = phf_map! {
     "node_modules"        => Icons::FOLDER_NPM,     // 
     "npm_cache"           => Icons::FOLDER_NPM,     // 
     "pam.d"               => Icons::FOLDER_KEY,     // 󰢬
+    "portage"             => '\u{f30d}',            // 
     "Pictures"            => '\u{f024f}',           // 󰉏
+    "root"                => '\u{f0250}',           // 󰉐
+    "src"                 => '\u{f19fc}',           // 󱧼
     "ssh"                 => Icons::FOLDER_KEY,     // 󰢬
     "sudoers.d"           => Icons::FOLDER_KEY,     // 󰢬
+    "vendor"              => '\u{f0ae6}',           // 󰫦
```

Comment:
How is vendor specific to Symfony?

Reply:
Symfony is not as associated with Composer as I once thought.